server/routes/shared/showReferralPresenter.test.ts

```diff
@@ -20,6 +21,8 @@ describe(ShowReferralPresenter, () => {
     serviceCategories: cohortServiceCategories,
   })
 
+  const deliusConviction = deliusConvictionFactory.build()
```

Comment:
We should be specific with the fields that affect the output of the class under test, and not rely on the factory's default values.

Reply:
Good point - does this apply to the above too, or only in unit tests? You've only mentioned `expectedSentenceEndDate`, but we also display `category` and `subcategory`, so wanted to double check.

Reply:
I think it should be in both, so I've added it to both and cut down the number of fields being explicitly set that we don't care about

Reply:
Yep, applies to the above. But in the above I was just talking about the sentence (the category and subcategory are on the offence, not the sentence).